components/profile/rating/src/index.scss

```diff
@@ -0,0 +1,30 @@
+@import '~@schibstedspain/theme-basic/lib/index';
+
+$fill-profile-rating-filled: #f7af33 !default;
+$fill-profile-rating-empty: #d7dbdf !default;
+
+.sui-ProfileRating {
+  height: 16px;
+  margin-top: 7px;
```

Comment:
Margins and Paddings should be base8. Thera are a set of declarations [here](https://github.com/SUI-Components/sui-theme/blob/master/src/settings/_spacing.scss) you should use instead